sedkit/relations.py

```diff
@@ -286,7 +286,7 @@ def evaluate(self, rel_name, x_val, xunits=None, yunits=None, fit_local=False, p
                 self.relations[rel_name] = full_rel
 
                 if y_upper is not None:
-                    return y_val, y_upper, y_lower, self.ref
+                    return y_val.round(3), y_upper.round(3), y_lower.round(3), self.ref
```

Comment:
I don't think you should round these values. instead, I think you should consider using %g formatting when displaying it to the user.
Check out the examples here: https://stackoverflow.com/a/3411731/4842634